pkg/splunk/enterprise/configuration.go

```diff
@@ -1151,6 +1173,17 @@ func ValidateAppFrameworkSpec(appFramework *enterpriseApi.AppFrameworkSpec, appC
 		appContext.AppsRepoStatusPollInterval = splcommon.MaxAppsRepoPollInterval
 	}
 
+	if appContext.AppsStatusMaxConcurrentAppDownloads <= 0 {
+		scopedLog.Info("Invalid value of maxConcurrentAppDownloads", "configured value", appContext.AppsStatusMaxConcurrentAppDownloads, "Defaulting to 5.")
+		appContext.AppsStatusMaxConcurrentAppDownloads = 5
```

Comment:
Should we make this a const instead of hardcoding a 5 here?

Reply:
Yup, right good point!